ubuntu/ubuntu.go

```diff
@@ -19,11 +19,19 @@ import (
 )
 
 const (
-	repoURL   = "https://git.launchpad.net/ubuntu-cve-tracker"
 	ubuntuDir = "ubuntu"
 )
 
 var (
+	repoURLs = []string{
+		"https://git.launchpad.net/ubuntu-cve-tracker",
+		"git://git.launchpad.net/ubuntu-cve-tracker",
+	}
+	targets = []string{
+		"active",
+		"ignored",
```

Comment:
If we delete all advisories once, we don't have to take `ignored` into consideration, right?

Reply:
yes I believe so